powerline/listers/i3wm.py

```diff
@@ -0,0 +1,81 @@
+# vim:fileencoding=utf-8:noet
+from __future__ import (unicode_literals, division, absolute_import, print_function)
+
+import re
+
+from powerline.theme import requires_segment_info
+from powerline.lib.shell import run_cmd
+
+
+conn = None
+
+
+def update_segment_info(segment_info, **kwargs):
```

Comment:
Function has absolutely nothing to do with segment_info. Better place this in `powerline.lib.dict`:

```Python


def updated(d, *args, **kwargs):
	'''Copy dictionary and update it with provided arguments
	'''
	d = d.copy()
	d.update(*args, **kwargs)
	return d
```

: I use this function in `editors-refactor` branch.